marketplace.Makefile

```diff
@@ -11,6 +11,20 @@ include $(makefile_dir)/common.Makefile
 	mkdir -p "$@"
 
 
+.build/marketplace/dev: $(MARKETPLACE_TOOLS_PATH)/marketplace/deployer_util/* \
+                        $(MARKETPLACE_TOOLS_PATH)/marketplace/dev/* \
+                        $(MARKETPLACE_TOOLS_PATH)/scripts/* \
```

Comment:
this doesn't include driver, maybe we should use `$(shell find)` here.

Reply:
I'm actually inclined to flatten that in the future. For now I'll include `$(MARKETPLACE_TOOLS_PATH)/scripts/driver/*` directly.